src/lib.rs

```diff
@@ -347,6 +346,8 @@ impl<'a, 'b> Builder<'a, 'b> {
     /// # Platform Notes
     /// ## Unix
     ///
+    /// Tempfiles default to `0o600` and temdirs default to `0o777`.
```

Comment:
This should be combined with the next sentence for clarity and should explain how the umask is combined. The reader needs to walk away understanding that, by default, the permissions on a directory are `0o755` given the standard `0o022` umask, etc.

Also, it should mention `Builder::permissions`.

Reply:
Oh, wait. This _is_ `Builder::permissions()`. Never mind that part.

But I still want the reader to come away with an understanding that their temporary directory is probably _not_ going to be world-writable.

Reply:
I tried to incorporate this, but further notes on it would be welcome.

Reply:
That's plenty good enough.